docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -428,14 +428,26 @@ This section describes how to use the [Supervisor API](../../api-reference/super
 
 ### AWS authentication
 
-To authenticate with AWS, you must provide your AWS access key and AWS secret key using `runtime.properties`, for example:
+Druid uses AWS access and secret keys to authenticate Kinesis API requests. There are a few ways to provide this information to Druid:
 
-```text
+1. Using roles or short-term credentials:
```

Comment:
```suggestion
1. Using roles or short-term credentials. 
```
Lines 435-438 do not align with line 433. Consider indenting the first line of the paragraph to align it with line 433. The same suggestion applies to lines 440 and 442.

Reply:
thanks, done!